fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/expressions/Field.java

```diff
@@ -64,6 +64,17 @@ public OneOfThem oneOfThem() {
         return new OneOfThem(fieldName);
     }
 
+    /**
+     * If the associated field is a repeated one, this allows you to match against one of those values, the record will
+     * be returned if any one of the values returns true. The record may be returned more than once.
+     * @param emptyIsUnknown {@code true} if an empty repeated field should cause an UNKNOWN result instead of failing to match any (and so returning FALSE).
```

Comment:
```suggestion
     * @param emptyIsUnknown {@code true} if an empty repeated field should cause an UNKNOWN result instead of failing to match any (and so returning FALSE)
```